terraform/network.tf

```diff
@@ -0,0 +1,56 @@
+resource "aws_vpc" "vpc" {
```

Comment:
Can we just not just go with a security group and since this is just temporary

Reply:
As per @v-y-a-s suggestion. We deployed it our sandboxed env and this will be destroyed.

Reply:
This might take a bit of time to setup but nevertheless, a good exercise. 

We will be using LZ2's VPC when we get access